doc/qos_overview.md

```diff
@@ -52,15 +59,15 @@ to virtual output queues that are instantiated for remote interfaces. In this
 case, statistics for each egress interface are reported within the ingress
 interface's entry in the `/qos/interfaces/interface` list.
 
-# Annotated QoS Example
+## Annotated QoS Examples
 
-## Ingress Classification with Egress Scheduling
+### Ingress Classification with Egress Scheduling
 
 The example QoS configuration below shows the configuration of an interface,
 assumed to be facing a customer which has ingress classification based on DSCP
 markings. The same interface has an egress scheduler policy applied to it.
```

Comment:
Can we expand a bit on what this achieves? i.e., there's an egress scheduler policy but it seems like it's material to the example that this is being used to police traffic to some expected output data rate.

Reply:
Oh, I did not intend this example to be related to  the ingress policer example below.